src/packages/carpool-view-flow/react/screens/trip-form/TripFormScreen.jsx

```diff
@@ -321,13 +271,23 @@ TripForm.propTypes = {
   stops: React.PropTypes.array.isRequired,
   fromAddress: React.PropTypes.string,
   toAddress: React.PropTypes.string,
+  locationFrom: React.PropTypes.string,
+  locationTo: React.PropTypes.string,
+  tripDateTime: React.PropTypes.object.isRequired,
+  dispatch: React.PropTypes.func.isRequired,
 }
 
+const connectedTripForm = connect((state) => ({
+  locationFrom: locationFromSelector(state),
+  locationTo: locationToSelector(state),
+  tripDateTime: tripDateTimeSelector(state),
+}))(TripForm);
+
 export default createContainer(() => {
   const progress = new Progress();
   const stops = carpoolService.pullStops(progress.setProgress.bind(progress, 'stops'));
```

Comment:
I should add some comments here - stops are/were needed to be loaded as path routing was done in client.
But I think it could be removed from TripForm.propTypes
In general those stops are required only when creatingDrive